src/java.base/share/classes/java/lang/StringLatin1.java

```diff
@@ -608,7 +602,8 @@ public static String toUpperCase(String str, byte[] value, Locale locale) {
         // Now check if there are any characters that need to be changed, or are surrogate
```

Comment:
Can you fix this one too for consistency? I'll sponsor either way after some testing.